src/scripts/ci.zig

```diff
@@ -97,3 +97,44 @@ fn build_docs(shell: *Shell) !void {
     try shell.exec("npm install", .{});
     try shell.exec("npm run build", .{});
 }
+
+fn validate_release(shell: *Shell, gpa: std.mem.Allocator, language_requested: ?Language) !void {
+    var tmp_dir = std.testing.tmpDir(.{});
+    defer tmp_dir.cleanup();
+
+    try tmp_dir.dir.setAsCwd();
+
+    const release_info = try shell.exec_stdout(
+        "gh release list --repo tigerbeetle/tigerbeetle --limit 1",
+        .{},
+    );
+    const tag = stdx.cut(release_info, "\t").?.prefix;
+    log.info("validateing release {s}", .{tag});
+
+    try shell.exec("gh release download {tag}", .{ .tag = tag });
+
+    if (builtin.os.tag != .linux) {
+        log.warn("skip release verification for platforms other than Linux", .{});
+    }
+
+    try shell.exec("unzip tigerbeetle-x86_64-linux.zip", .{});
+    const version = try shell.exec_stdout("./tigerbeetle version --verbose", .{});
+    assert(std.mem.indexOf(u8, version, tag) != null);
+    assert(std.mem.indexOf(u8, version, "ReleaseSafe") != null);
+
+    const tigerbeetle_absolute_path = try std.fs.cwd().realpathAlloc(gpa, "tigerbeetle");
+    defer gpa.free(tigerbeetle_absolute_path);
+
+    inline for (comptime std.enums.values(Language)) |language| {
+        // TODO: these two don't work yet
+        if (language == .go or language == .dotnet) continue;
```

Comment:
Yes, these two are broken! Looking into fixing them, but want to do that independently from this work to add basic verification.